### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by GabrielleCasimiro**
Minor changes in documentation : typos, small reformulations, links, ...

### CODE REVIEW

**Comment by GabrielleCasimiro** [comment]
Attention : In [docs/get_started.md](https://github.com/GabrielleCasimiro/leaspy/blob/gc/documentation/docs/get_started.md), the link to the Quickstart example does not work.

File: docs/index.md [hunk 1]
```diff
@@ -37,19 +37,18 @@ The main documentation. This contains an in-depth description of all algorithms
 
 #### License
 
-The package is distributed under the GNU LESSER GENERAL PUBLIC LICENSE.
+The package is distributed under the BSD 3-Clause license.
```
Comment (KaisaridiSofia) [inline docs/index.md]:
```suggestion
The package is distributed under the BSD-3 license.
```

File: docs/algorithms.md [hunk 2]
```diff
@@ -2,54 +2,39 @@
...
+We need to specify the arguments `name`, `dimension` (the number of outcomes $K$ in your dataset) and the `obs_models` (valid choices for the logistic model are 'gaussian-diagonal' to estimate one noise coefficient per outcome or 'gaussian-scalar' to estimate one noise coefficient for all the outcomes). When we fit a multivariate model we also need to specify `source_dimension` that corresponds to the degrees of freedom of intermarker spacing parameters. We refer you to the [mathematical background section](./mathematics.md#individual-trajectory--spatial-random-effects) for more details. We generally suggest a number of sources close to the square root of the number of outcomes ($\sqrt{dimension}$).
+
+You can also add a `seed` or control other arguments for the output and the logs like `save_periodicity`, `path`, etc.
 
 ```python
 model = LogisticModel(name="my-model", source_dimension=1, dimension=2, obs_models='gaussian-diagonal')
 model.fit(data_leaspy, "mcmc_saem", n_iter=20000)
```
Comment (MendezSebastianP) [inline docs/algorithms.md]:
```suggestion
model.fit(data_leaspy, "mcmc_saem", n_iter=20000, seed=42)
```

File: docs/index.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 ## LEArning Spatiotemporal Patterns in Python
 
 ### Description
 
 **Leaspy** is a software package for the statistical analysis of **longitudinal data**, particularly **medical** data that comes in a form of **repeated observations** of patients at different time-points.
 
-![leaspy front](./_static/images/leaspy_front.png)
+![leaspy_front](./_static/images/leaspy_front.png)
+
+*Figure 1: Illustration of the Disease Progression Model underlying Leaspy. (a) illustrates how sparse individual observations (colored dots) can be recombined into a common population trajectory (in black), while accounting for the fact that each subject may progress earlier/later or faster/slower than the average (the green individual progresses earlier, and the blue one slower than the average). (b) shows the corresponding geometric interpretation: individual data points are mapped onto a shared trajectory on a Riemannian manifold. Individual trajectories are then computed through subject-specific temporal transformations (time-shift $\tau_i$ and acceleration $\alpha_i$), and spatial transformations ($w_i$) capturing differences in the ordering and pattern of deterioration across outcomes.*
```
Comment (MendezSebastianP) [inline docs/index.md]: Putting in the right format

File: docs/install.md
Comment (MendezSebastianP) [inline docs/install.md]:
```suggestion
There are multiple solutions for that, the most common option is to use `conda`:
```

File: docs/install.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ cd leaspy
 This project relies on [poetry](https://python-poetry.org) that you would need to install (see the [official instructions](https://python-poetry.org/docs/#installation)).
 
 ```{tip}
-It is recommended install it in a dedicated environment, separated from the one in which you will install Leaspy and its dependencies. One possibility is to install it with a tool called [pipx](https://pipx.pypa.io/stable/).
+It is recommended to install it in a dedicated environment, separated from the one in which you will install Leaspy and its dependencies. One possibility is to install it with a tool called [pipx](https://pipx.pypa.io/stable/).
 
 If you don't have `pipx` installed, already, you can follow the [official installation guidelines](https://pipx.pypa.io/stable/installation/).
```
Comment (MendezSebastianP) [inline docs/install.md]:
```suggestion
If you don't have `pipx` installed already, you can follow the [official installation guidelines](https://pipx.pypa.io/stable/installation/).
```

File: docs/install.md
Comment (MendezSebastianP) [inline docs/install.md]:
```suggestion
To do so, in your `leaspy` environment, you can download ``ipykernel`` to use `leaspy` with `jupyter` notebooks:
```

**Review by MendezSebastianP — CHANGES_REQUESTED** [review]
Great job, @GabrielleCasimiro, with all these pertinent changes. I'm leaving some minor suggestions for you. Some of them aren't part of your original corrections, but are just small improvements I noticed while reviewing the code.

File: docs/algorithms.md [hunk 5]
```diff
@@ -2,54 +2,39 @@
...
+We need to specify the arguments `name`, `dimension` (the number of outcomes $K$ in your dataset) and the `obs_models` (valid choices for the logistic model are 'gaussian-diagonal' to estimate one noise coefficient per outcome or 'gaussian-scalar' to estimate one noise coefficient for all the outcomes). When we fit a multivariate model we also need to specify `source_dimension` that corresponds to the degrees of freedom of intermarker spacing parameters. We refer you to the [mathematical background section](./mathematics.md#individual-trajectory--spatial-random-effects) for more details. We generally suggest a number of sources close to the square root of the number of outcomes ($\sqrt{dimension}$).
+
+You can also add a `seed` or control other arguments for the output and the logs like `save_periodicity`, `path`, etc.
 
 ```python
```
[the remark this answers is not in the document]
Comment (KaisaridiSofia) [reply docs/algorithms.md]: I think it's not really a big deal here. We can leave it like that, just mention that these arguments exist as well and it's ok if we don't use them directly after. If the user wants more details on how to use these arguments I think they can be found in the docstrings of the fitting algorithm.

**Comment by GabrielleCasimiro** [comment]
> Thank you @MendezSebastianP for the review. I think all of your comments are to the point and we can commit the suggestions. If @GabrielleCasimiro is ok as well I think we can commit all the changes and merge.

Thanks @MendezSebastianP for the review ! Everything seems good to me too, I think we can add all the suggestions and merge.

**Comment by MendezSebastianP** [comment]
Last changement i forgot, could you change the file pyproject.toml in line 7:
version = "2.0.0"

thanks!
